{"name": "test_simple", "status": "failed", "statusDetails": {"message": "AssertionError: assert 4 == 5", "trace": "self = <test_login_page.TestLoginPage object at 0x7fb8efd82090>\n\n    def test_simple(self):\n        self.login_page.open()\n        self.login_page.enter_login(\"123\")\n>       assert 4 == 5\nE       assert 4 == 5\n\ntests/test_login_page.py:24: AssertionError"}, "start": 1745087438567, "stop": 1745087439582, "uuid": "7d95b139-6c7b-4dc8-b09a-44c4c42d22a5", "historyId": "b625b33df7b710c19b0b63ad85e53788", "testCaseId": "b625b33df7b710c19b0b63ad85e53788", "fullName": "tests.test_login_page.TestLoginPage#test_simple", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_page"}, {"name": "subSuite", "value": "TestLoginPage"}, {"name": "host", "value": "4d3ca183eccd"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_page"}]}